ruma-events/src/room/redaction.rs

```diff
@@ -55,93 +55,33 @@ pub struct RedactionEventStub {
     pub unsigned: UnsignedData,
 }
 
+/// A redacted redaction event.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[ruma_event(type = "m.room.redaction")]
+#[ruma_event(custom_redacted)]
+pub struct RedactedRedactionEventContent;
+
+impl ruma_events::RoomEventContent for RedactedRedactionEventContent {}
+
+impl ruma_events::MessageEventContent for RedactedRedactionEventContent {}
+
 /// A redaction of an event.
 #[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
 #[ruma_event(type = "m.room.redaction")]
+#[ruma_event(custom_redacted)]
 pub struct RedactionEventContent {
     /// The reason for the redaction, if any.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub reason: Option<String>,
 }
 
+impl RedactionEventContent {
+    /// Transforms the full event content into a redacted content according to spec.
+    pub fn redact(self) -> RedactedRedactionEventContent {
+        RedactedRedactionEventContent
+    }
+}
+
 impl ruma_events::RoomEventContent for RedactionEventContent {}
 
 impl ruma_events::MessageEventContent for RedactionEventContent {}
-
```

Comment:
Why did you remove these tests?

Reply:
They were the same as in `tests/redaction.rs`